### PR TITLE
Refactor: remove events tags requirements from fields and resolvers

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

There is no reason to use eventTags every time we try to use Event, so this aims to separate them.

### CODE REVIEW

File: server/prisma/generator/factories/events.factory.ts [hunk 1]
```diff
@@ -83,52 +83,6 @@ const createEvents = async (
...
-    await prisma.events.update({
-      where: { id: event.id },
-      data: {
-        tags: {
-          create: connectOrCreateTags,
-        },
-      },
-    });
-
-    events.push(event.id);
```
Comment (Sboonny) [inline server/prisma/generator/factories/events.factory.ts:131]: I don't know why I have deleted factory, I guess I was on frenzy. anyway should they be brought back or when we try to use EventTags in the future we will use them?

File: server/src/controllers/Events/resolver.ts [hunk 2]
```diff
@@ -719,22 +700,11 @@ ${unsubscribeOptions}`,
...
-              tag: {
-                connectOrCreate: {
-                  create: { name: tagName },
-                  where: { name: tagName },
-                },
-              },
-            })),
-          },
-        },
+        data: {},
```
Comment (Sboonny) [inline server/src/controllers/Events/resolver.ts:707]: Shouldn't these be deleted as well? I amn't sure, but now they don't update anything

File: server/src/graphql-types/Event.ts [hunk 3]
```diff
@@ -44,10 +44,14 @@ export class Event extends BaseObject {
...
   @Field(() => String)
   image_url: string;
+}
 
+@ObjectType()
+export class EventWithTag extends Event {
   @Field(() => [EventTag])
   tags: EventTag[];
 }
+
```
Comment (ojeytonwilliams) [inline server/src/graphql-types/Event.ts:54]:
```suggestion
```
This doesn't get used, so we can remove it.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Looking good, just one minor thing:

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**